hydra-node/test/Hydra/BehaviorSpec.hs

```diff
@@ -831,9 +861,8 @@ toOnChainTx now = \case
   FanoutTx{} ->
     OnFanoutTx{headId = testHeadId}
 
--- NOTE(SN): Deliberately long to emphasize that we run these tests in IOSim.
```

Comment:
I think this still holds and we should keep it long (one hour is not long, this might rather be a couple days at some point)

Reply:
I got the test timeouts because now contestation happens now in _can close with decommit in flight_ spec. Perhaps the deadline calculation is somehow wrong and causing too long contestations?